controllers/apps/components/component.go

```diff
@@ -389,6 +389,17 @@ func (c *rsmComponent) status(reqCtx intctrlutil.RequestCtx, cli client.Client,
 		return err
 	}
 
+	// when the component is ready for the first time, execute a custom postStart hook once.
```

Comment:
"the component is ready, try to execute...", there is no first time or once here